test/aqua.jl

```diff
@@ -0,0 +1,36 @@
+using Test
+using StatsBase
+using Aqua
+
+@testset "Aqua tests (performance)" begin
+    # This tests that we don't accidentally run into
+    # https://github.com/JuliaLang/julia/issues/29393
+    # Aqua.test_unbound_args(StatsBase)
+    ua = Aqua.detect_unbound_args_recursively(StatsBase)
+    @test length(ua) == 0
```

Comment:
Can we just run the full Aqua test suite (see also my other comment below)? It's already included there by default: https://github.com/JuliaTesting/Aqua.jl/blob/3d5ed9fa2c915bbb06b2ef9037d57029d18bc8b5/src/unbound_args.jl#L36

Reply:
Yeah, I guess we can change it to the test function. I have my own gripe with that interface, tbh. https://github.com/JuliaTesting/Aqua.jl/issues/180